# Custom ROS 2 message for 3D grid
uint32 width
uint32 height
uint32 depth
uint64[] data
